### DISCUSSION TITLE
Deceptive Site Ahead

### DISCUSSION THREAD

**Discussion by sudoshivesh** [community/community#53815] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Whenever I am trying to open any page from my github via link it is showing deceptive page ahead

**Comment by airtower-luna** [comment]
It looks like your domain (`sudoshivesh.github.io`) ended up on the Google Safe Browsing block list, reported for phishing. This has nothing to do with GitHub.

The question is: Why? I've read of people getting hit by that for putting up pages that imitate popular websites. Even if the intention is just practice, that can end up looking deceptive.

If you have anything like that, you'll need to remove it to get off the block list, and then request your site to be removed. See: https://developers.google.com/search/docs/monitor-debug/security/social-engineering#fixing-the-problem (point 2)

**Comment by Ahmedghazi1981** [comment]
Yes i am see him in the hosptal 2019

في خميس، ٢٧ أبريل، ٢٠٢٣ في ١٢:٢٨ ص، كتب Airtower ***@***.***>:

> It looks like your domain (sudoshivesh.github.io) ended up on the Google
> Safe Browsing block list, reported for phishing. This has nothing to do
> with GitHub.
>
> The question is: Why? I've read of people getting hit by that for putting
> up pages that imitate popular websites. Even if the intention is just
> practice, that can end up looking deceptive.
>
> If you have anything like that, you'll need to remove it to get off the
> block list, and then request your site to be removed. See:
> https://developers.google.com/search/docs/monitor-debug/security/social-engineering#fixing-the-problem
> (point 2)
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>
-- 
𒀀𒄩𒈠𒁕
